lib/resources/majors.tsx

```diff
@@ -5077,6 +5330,180 @@ const minorMath: Minor = {
+            'AS.110.106 Calculus I (Biology and Social Sciences)' +
+            '<br /> <i>OR</i> <br />' +
+            'AS.110.108 Calculus I (Physical Sciences and Engineering)',
+          required_credits: 4,
+          criteria: 'AS.110.106[C]^OR^AS.110.108[C]',
+        },
+        {
+          description:
+            '<b>Calculus II</b> <br />' +
+            'AS.110.109 Calculus II (For Physical Sciences and Engineering)' +
+            '<br /> <i>OR</i> <br />' +
+            'AS.110.107 Calculus II (For Biological and Social Science)',
+          required_credits: 4,
+          criteria: 'AS.110.109[C]^OR^AS.110.107[C]',
+        },
+        {
+          description:
+            '<b>Calculus III</b> <br /> AS.110.202 Calculus III' +
+            '<br /> <i>OR</i> <br /> AS.110.211 Honors Multivariable Calculus',
+          required_credits: 4,
+          criteria: 'AS.110.202[C]^OR^AS.110.211[C]',
+        },
+        {
+          description:
+            '<b>Linear Algebra</b> <br /> AS.110.201 Linear Algebra<br /> <i>OR</i> ' +
+            '<br /> EN.553.291 Linear Algebra and Differential Equations<br /> <i>OR</i> ' +
+            '<br /> AS.110.212 Honors Linear Algebra',
+          required_credits: 4,
+          criteria: 'AS.110.201[C]^OR^EN.553.291[C]^OR^AS.110.212[C]',
+        },
+      ],
+    },
+    {
+      name: 'Fundamental Computer Integrated Surgery Courses',
+      required_credits: 8,
+      min_credits_per_course: 4,
+      description:
+        'You must take Computer Integrated Surgery I and a design course in CIS.',
+      criteria:
+        '(EN.510.433[C]^OR^EN.510.434[C])^OR^(EN.520.462[C]^OR^EN.520.463[C])^OR^' +
+        '(EN.520.498[C]^OR^EN.520.499[C])^OR^(EN.540.400[C]^OR^EN.540.421[C])^OR^' +
+        '(EN.580.311[C]^OR^EN.580.312[C])^OR^(EN.580.411[C]^OR^EN.580.412[C])^OR^' +
+        '(EN.580.456[C]^OR^EN.580.457[C])^OR^(EN.580.471[C]^OR^EN.580.571[C])^OR^' +
+        '(EN.580.480[C]^OR^EN.580.481[C])^OR^(EN.580.580[C]^OR^EN.580.581[C])^OR^' +
+        'EN.601.455[C]^OR^EN.601.456[C]^OR^(EN.580.437[C]^OR^EN.580.438[C])',
+      fine_requirements: [
+        {
+          description: '<b>EN.601.455 Computer Integrated Surgery I</b>',
+          required_credits: 4,
+          criteria: 'EN.601.455[C]',
+        },
+        {
+          description:
+            '<b>EN.601.456 Computer Integrated Surgery II or design course in CIS</b>',
+          required_credits: 4,
+          criteria:
+            '(EN.510.433[C]^OR^EN.510.434[C])^OR^(EN.520.462[C]^OR^EN.520.463[C])^OR^' +
+            '(EN.520.498[C]^OR^EN.520.499[C])^OR^(EN.540.400[C]^OR^EN.540.421[C])^OR^' +
+            '(EN.580.311[C]^OR^EN.580.312[C])^OR^(EN.580.411[C]^OR^EN.580.412[C])^OR^' +
+            '(EN.580.456[C]^OR^EN.580.457[C])^OR^(EN.580.471[C]^OR^EN.580.571[C])^OR^' +
+            '(EN.580.480[C]^OR^EN.580.481[C])^OR^(EN.580.580[C]^OR^EN.580.581[C])^OR^' +
+            'EN.601.456[C]^OR^(EN.580.437[C]^OR^EN.580.438[C])',
```

Comment:
do these include design courses not in CIS? uwu

A design course in CIS. Either EN.601.456 Computer Integrated Surgery II or a design course in biomedical engineering, electrical and computer engineering, or mechanical engineering with substantial CIS content approved by the student’s faculty advisor in the CIS minor.

Reply:
also is there a better way to query this like 

if design is a tag you could do design tag + department